pkg/executor/runner/runner.go

```diff
@@ -299,23 +303,25 @@ func (r *Runner) WriteConfigFile() error {
 		return err
 	}
 
-	err = writer.Write(jsonConfig, "/root/module/config.tf.json")
+	err = writer.Write(jsonConfig, fmt.Sprintf("%s/config.tf.json", path))
 	if err != nil {
 		return err
 	}
 
+	logrus.Infof("Wrote backend config ", fmt.Sprintf("%s/config.tf.json", path))
 	return nil
 }
 
-func (r *Runner) WriteVarFile() error {
+func (r *Runner) WriteVarFile(path string) error {
 	vars, ok := r.VarSecret.Data["varFile"]
 	if !ok {
 		return fmt.Errorf("no varFile data found in secret %v", r.VarSecret.Name)
 	}
-	err := writer.Write(vars, fmt.Sprintf("/root/module/%v.auto.tfvars.json", r.Execution.Name))
+	err := writer.Write(vars, fmt.Sprintf("%s/%v.auto.tfvars.json", path, r.Execution.Name))
```

Comment:
Please use `path.Join()` here instead of concatenation.

Reply:
done